cnxepub/html_parsers.py

```diff
@@ -16,6 +16,15 @@
     }
 
 
+def _squash_to_text(elm):
+    value = [elm.text or '']
+    for child in elm.getchildren():
+        value.append(etree.tostring(child).decode('utf-8'))
+        value.append(child.tail or '')
+    value = ''.join(value).encode('utf-8')
+    return value
+
+
```

Comment:
>>> e.text=u'This is élément'
>>> etree.tostring(e1)
'<div><a>This is &#233;l&#233;ment</a></div>'
>>> etree.tostring(e1, method='html', encoding='utf-8')
'<div><a>This is \xc3\xa9l\xc3\xa9ment</a></div>'

One of those should work, no?